### PR TITLE
Added container to body

### LINKED ISSUE #29: Enhancement:  place body in container
### Describe the Current Behavior/Feature:

currently all page content has no container to limit max width

---

### Proposed Implementation:

Add container over main content in layout and fix bugs that it causes.

### PR DESCRIPTION
**Pull request by Jared-Krajewski**
kinda a bug kinda an enhancement either way theres a body around the container now and i fixed a width issue with volunteer brief to make it center again.

## This PR:

Resolves #29 

## Screenshots (if applicable):

Screenshot 2023-10-25 at 2 13 17 PM

### CODE REVIEW

**Comment by andycwilliams** [comment]
The only issue I can see right now is that the width for the hero image is now constricted.

**Comment by Jared-Krajewski** [comment]
> The only issue I can see right now is that the width for the hero image is now constricted.

It was intentionally within the container. I guess its done both ways depending on preference. I changed it do exclude the hero on each page.

File: src/pages/Home.jsx [hunk 1]
```diff
@@ -1,20 +1,22 @@
-import Hero from "../components/home/Hero";
-import About from "../components/home/About";
-import VolunteerBrief from "../components/home/VolunteerBrief";
-import Events from "../components/home/Events";
-import ProjectsBrief from "../components/home/ProjectsBrief";
-import Partners from "../components/home/partners/Partners";
+import Hero from '../components/home/Hero';
```
Comment (andycwilliams) [inline src/pages/Home.jsx:4]: We can add `// Component Imports` here

File: src/pages/Home.jsx [hunk 2]
```diff
@@ -1,20 +1,22 @@
...
+import About from '../components/home/About';
+import VolunteerBrief from '../components/home/VolunteerBrief';
+import Events from '../components/home/Events';
+import ProjectsBrief from '../components/home/ProjectsBrief';
+import Partners from '../components/home/partners/Partners';
+import { Container } from '@mui/material';
```
Comment (andycwilliams) [inline src/pages/Home.jsx]: And then add `// Material UI Imports` here. And so far we've been putting these imports above the component ones.

**Review by andycwilliams — APPROVED** [review]
Looks good. My only comments are nitpicky stuff.

Also, I don't know if we'll want to set up an `index.js` file for subfolders with multiple files (such as `partners`). It's what we do for PASS. It doesn't harm us not to, it just might make things a bit cleaner.

**Comment by Jared-Krajewski** [comment]
> Also, I don't know if we'll want to set up an `index.js` file for subfolders with multiple files (such as `partners`). It's what we do for PASS. It doesn't harm us not to, it just might make things a bit cleaner.

made commented changes. I'm not sure thats necessary due to this being a significantly smaller project. Also not sure its necessary to even label the imports, their naming is pretty explanatory for where they come from.  Not sure why its done in most of PASS either.

**Pull request merged**